src/sagas/save-project-settings.saga.js

```diff
@@ -71,6 +73,7 @@ export function* handleSaveSettings(action: any): Saga<void> {
     json = yield call(loadPackageJson, projectPath);
   } catch (err) {
     yield call(handleProjectSaveError, new Error('loading-packageJson-failed'));
+    return;

```

Comment:
I noticed that even if the error was thrown, the process tried to continue, which led to another error. So instead I bailed out after the first error